frontend/src/components/element/Breadcrumb.tsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from 'react-router-dom';
+
+/**
+ * BreadcrumbItem interface defines the properties for each item in the Breadcrumb component.
+ *  label: string - The text to be displayed for the breadcrumb item
+ *  path?: string - Optional path used for navigation; if not provided, item is displayed as plain text
+ */
+export interface BreadcrumbItem {
+  label: string;
+  path?: string;
+}
+
+/**
+ * BreadcrumbProps interface defines the properties for the Breadcrumb component.
+ *  items: BreadcrumbItem[] - An array of breadcrumb items to be rendered in order
+ */
+interface BreadcrumbProps {
+  items: BreadcrumbItem[];
+}
+
+/**
+ * Breadcrumb component that displays a list of navigation links.
+ * Follows BC Gov design system principles to help users understand
+ * their current location within the site structure.
+ *
+ * @param items - Array of BreadcrumbItem objects with label and optional path
+ * @returns JSX.Element - The rendered Breadcrumb navigation component
+ */
+export const Breadcrumb = ({ items }: BreadcrumbProps) => {
+  return (
+    <div className="w-full">
+      {/* Breadcrumb items below mirror the grid layout defined in BaseLayout component */}
+      <div className="grid grid-cols-8 gap-2">
+        <div className="grid sm:col-start-1 sm:col-end-9 md:col-start-2 md:col-end-8 lg:col-start-3 lg:col-end-7">
+          <div className="py-[14px]">
+            <nav className="text-[18px]" aria-label="Breadcrumb">
+              <ol className="flex flex-wrap items-center space-x-[8px]">
```

Comment:
Just to decrease the nesting a little the class on line 35 can be added to line 37.  (note with change below the corresponding </div> will have to be removed from the bottom of this file 
```suggestion
            <nav className="text-[18px]" aria-label="Breadcrumb">
              <ol className="flex flex-wrap items-center py-[14px] space-x-[8px]">
```